packages/react-components/react-combobox/src/components/Listbox/useListbox.ts

```diff
@@ -90,6 +89,27 @@ export const useListbox_unstable = (props: ListboxProps, ref: React.Ref<HTMLElem
     }
   };
 
+  const onFocus = (_event: React.FocusEvent<HTMLElement>) => {
+    // set initial active descendent upon focus, in the absence of a parent context controlling the state
+    if (!hasParentActiveDescendantContext) {
```

Comment:
one extremely minor nit is that it's sometimes a little easier to read this, since it results in fewer levels of brackets:

```javascript
if (!hasParentActiveDescendantContext) {
  return;
}

// rest of code at the same indent
```

Reply:
I agree, fixed :)